src/SprykerEco/Zed/Adyen/Business/Handler/Redirect/OnlineTransferRedirectHandler.php

```diff
@@ -85,6 +85,15 @@ public function handle(AdyenRedirectResponseTransfer $redirectResponseTransfer):
             return $redirectResponseTransfer;
         }
 
+        $resultCodeLower = strtolower($responseTransfer->getPaymentDetailsResponseOrFail()->getResultCode());
+        if ($resultCodeLower === $this->config->getAdyenPaymentStatusRefused()) {
```

Comment:
`$this->config->getAdyenPaymentStatusRefused()` call result could be moved to a variable to not call it twice